### DISCUSSION TITLE
RANDOM FOREST ALGORITHM

### DISCUSSION THREAD

**Discussion by duhsoaresz** [semiautomaticgit/SemiAutomaticClassificationPlugin#228] [Q&A] [answered]
Dear Luca Congelo.
How are you ?
I send this message for you  in order to solve some doubts about the Random Forest  algorithm .
First of all, I am an engineering student at Universidade Estadual Paulista in Brazil and we are going to use RF to develop the works.
continuing
My question is just one: What attributes does the algorithm collect, when we look for ROIs, variance? Standard deviation? Average ? If there is somewhere that explains this, please re-direct me.
Thanks in advance and I look forward to your answers.

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @duhsoaresz , the Random Forest tool is provided through SNAP software, therefore you should refer to SNAP documentation about the algorithm implementation.
